web/client/actions/map.js

```diff
@@ -5,16 +5,10 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree.
  */
-const assign = require('object-assign');
-const axios = require('axios');
 
 const CHANGE_MAP_VIEW = 'CHANGE_MAP_VIEW';
-const LOAD_FEATURE_INFO = 'LOAD_FEATURE_INFO';
-const ERROR_FEATURE_INFO = 'ERROR_FEATURE_INFO';
-const EXCEPTIONS_FEATURE_INFO = 'EXCEPTIONS_FEATURE_INFO';
-const CHANGE_MAPINFO_STATE = 'CHANGE_MAPINFO_STATE';
-const NEW_MAPINFO_REQUEST = 'NEW_MAPINFO_REQUEST';
-const PURGE_MAPINFO_RESULTS = 'PURGE_MAPINFO_RESULTS';
+const CLICK_ON_MAP = 'CLICK_ON_MAP';
+const CHANGE_MOUSE_POINTER = 'CHANGE_MOUSE_POINTER';
```

Comment:
We probably don't need an action to change the mouse pointer. The mouse pointer can be probably be a calculated property of existing state(it depends from controls state), but let's just open a new issue for this. It's ok for now. Actions should reflect events. I cannot figure out a real "change mouse pointer" event in this context.